### PR TITLE
CS fixes

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | o 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php [hunk 1]
```diff
@@ -41,8 +41,8 @@ public function __construct(ContainerInterface $container)
      *
      * @param string    $eventName The name of the event to dispatch. The name of the event is
```
Comment (xabbuh) [inline src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php]: Don't we then need to change the description here too?

Comment (fabpot) [reply src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php]: I agree, but out of the scope of this PR. I just want the code to have 0 CS errors. If someone wants to improve the current description, they are more than welcome, but I would not see that as a priority.

Comment (xabbuh) [reply src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php]: I agree, I was just wondering why the fixer complained about one case while leaving the other untouched.

**Comment by keradus** [comment]
Hi @fabpot , thanks for note.
This is bug that we had sadly introduced in last release of 2.12.

If description is a multiple sentences, rule shall not change it.

--------

This is good change:

```diff
-@param string $foo Description of param.
+@param string $foo description of param
```

This is not a good change:

```diff
-@param string $foo Description of param. Multiple sentences.
+@param string $foo description of param. Multiple sentences
```

----

This bug will be fixed in next 2.12 release, see PR to see state of work on that:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4027

**Comment by keradus** [comment]
This PR follows after tool, but it introduces bad changes, that won't happen anymore after fix on PHP CS Fixer side.
I would suggest to revert it.

**Pull request merged**